src/keylessBackup/PhoneBackupInput.tsx

```diff
@@ -0,0 +1,164 @@
+import { Countries } from '@celo/phone-utils'
+import { NativeStackScreenProps } from '@react-navigation/native-stack'
+import React, { useEffect, useMemo, useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import { SafeAreaView, ScrollView, StyleSheet, Text } from 'react-native'
+import * as RNLocalize from 'react-native-localize'
+import { useSelector } from 'react-redux'
+import { defaultCountryCodeSelector, e164NumberSelector } from 'src/account/selectors'
+import { KeylessBackupEvents } from 'src/analytics/Events'
+import ValoraAnalytics from 'src/analytics/ValoraAnalytics'
+import Button, { BtnSizes, BtnTypes } from 'src/components/Button'
+import PhoneNumberInput from 'src/components/PhoneNumberInput'
+import i18n from 'src/i18n'
+import Times from 'src/icons/Times'
+import { KeylessBackupFlow } from 'src/keylessBackup/types'
+import { emptyHeader } from 'src/navigator/Headers'
+import { navigate, navigateBack } from 'src/navigator/NavigationService'
+import { Screens } from 'src/navigator/Screens'
+import { TopBarIconButton } from 'src/navigator/TopBarButton'
+import { StackParamList } from 'src/navigator/types'
+import Colors from 'src/styles/colors'
+import fontStyles from 'src/styles/fonts'
+import { getPhoneNumberState } from 'src/verify/VerificationStartScreen'
```

Comment:
is there an easy common place we can pull this out? over importing directly from VerificationStartScreen?